src/Powercord/plugins/pc-announcements/Notice.jsx

```diff
@@ -1,32 +1,72 @@
 const { React, getModule, getModuleByDisplayName } = require('powercord/webpack');
 
 const Clickable = getModuleByDisplayName('Clickable');
-const classes = getModule([ 'noticeBrand' ]);
+
+let classesStore = null;
 
 const Notice = class Notice extends React.Component {
+  constructor () {
+    super();
+
+    this.state = classesStore || {
+      types: [],
+      button: '',
+      dismiss: ''
+    };
+  }
+
+  async componentDidMount () {
+    if (!classesStore) {
+      const classes = await getModule([ 'noticeBrand' ]);
+      Promise.all([ classes ])
+        .then(value => {
```

Comment:
This Promise.all() call seems unnecessary to me. `classes` is already an awaited variable, i.e. a resolved promise. You should be able to kill `Promise.all([ classes ]).then(value => { ... })` and replace all instances of `value[0]` with `classes`.

Reply:
That's a good call! Honestly not sure why I chose to use the `Promise.all()` call when there's only ever one key returned from the `noticeBrand` module. Anyway, I've gone ahead and added these requested changes to my latest commit.